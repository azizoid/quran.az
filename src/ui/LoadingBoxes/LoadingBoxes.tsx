import styles from './LoadingBoxes.module.scss'

export const LoadingBoxes = () => (
  <div className={styles.loadingBoxes}>
    <div className={styles.ldsFacebook}>
      <div></div>
      <div></div>
      <div></div>
    </div>
  </div>
)
