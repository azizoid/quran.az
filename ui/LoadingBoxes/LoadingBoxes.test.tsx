import { render } from '@testing-library/react'
import { LoadingBoxes } from './LoadingBoxes'

test('Render LoadingBoxes', () => {
  const { container } = render(<LoadingBoxes />)

  expect(container).toMatchSnapshot()
})
