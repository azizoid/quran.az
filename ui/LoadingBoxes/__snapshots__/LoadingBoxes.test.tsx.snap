// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render LoadingBoxes 1`] = `
<div>
  <div
    class="loadingBoxes"
  >
    <div
      class="ldsFacebook"
    >
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
